app/views/track/_tracking_links_simple.html.erb

```diff
@@ -6,10 +6,10 @@
 
 <% if existing_track %>
   <div class="feed_link feed_link_<%= location %>">
-    <%= link_to _("&ndash; Unfollow"), { :controller => 'track', :action => 'update', :track_id => existing_track.id, :track_medium => "delete", :r => request.fullpath }, :class => "link_button_green track-request-action" %>
+    <%= link_to _("Unfollow"), { :controller => 'track', :action => 'update', :track_id => existing_track.id, :track_medium => "delete", :r => request.fullpath }, :class => "link_button_green track-request-action" %>
   </div>
 <% elsif track_thing %>
   <div class="feed_link feed_link_<%= location %>">
-      <%= link_to _("&plus; Follow"), do_track_path(track_thing), :class => "link_button_green track-request-action" %>
+    <%= link_to _("Follow"), do_track_path(track_thing), :class => "link_button_green track-request-action" %>
```

Comment:
I've removed the +/- because I think they should be CSS icons. It doesn't make sense for `+` and `-` to be in the translation strings.